packages/react-app/src/services/giveawayparticipation.ts

```diff
@@ -28,14 +28,30 @@ const wonGiveaway = (giveaway: Giveaway, userInfo?: UserInfo) => {
   return giveaway.winners.some((winner) => winner.id === userInfo.email);
 };
 
+const hasPendingWinners = (giveaway: Giveaway, userInfo: UserInfo) => {
+  if (userInfo.role === UserRole.ADMIN) {
+    if (
+      giveaway.manual &&
+      new Date() > giveaway.endTime &&
+      giveaway.winners.length === 0
+    ) {
+      return true;
+    }
+  }
+  return false;
```

Comment:
Maybe the role validation should not be inside the function since that does not interfere on whether there are pending winners, only on the visibility of those pending winners